runtime/src/lib.rs

```diff
@@ -316,7 +316,8 @@ parameter_types! {
     pub const TechnicalCollectiveMotionDuration: BlockNumber = 5 * DAYS;
     pub const TechnicalCollectiveMaxProposals: u32 = 100;
     pub const TechnicalCollectiveMaxMembers: u32 = 100;
-    pub const SchedulerMaxWeight: Weight = Weight::from_parts(1024, 0);
+    pub const SchedulerMaxWeight: Weight = Perbill::from_percent(60) * BlockWeights::get().max_block;
```

Comment:
```suggestion
    pub const SchedulerMaxWeight: Weight = BlockWeights::get().max_block.saturating_mul(Perbill::from_percent(60));
```

Reply:
This method is not implemented for this types. And mul implementation for Perbill is working without overflows